gpbft/participant_test.go

```diff
@@ -83,10 +83,14 @@ func newParticipantTestSubject(t *testing.T, seed int64, instance uint64) *parti
 	return &subject
 }
 
-func (pt *participantTestSubject) expectBeginInstance() {
+func (pt *participantTestSubject) expectBeginInstance(fromFinality bool) {
```

Comment:
I expect these changes to revert when allowing the participant to fetch the committee as usual.